salt/kubelet/check-status.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+
+set -euo pipefail
+
+NODE_NAME="$1"
+
+##########################################################
+
+log() { echo "[kubelet-status]: $1 " ; logger -t "kubelet-status" "$1" ; }
+
+exit_check() {
+	log "$1"
+	exit 0
+}
+
+log "Checking node $NODE_NAME status"
+
+NODE_STATUS=`kubectl get node $NODE_NAME`
```

Comment:
Rather than checking via shelling out - should we use the kubelet health endpoint? i.e `http://127.0.0.1:10248/healthz` ?

Reply:
Ok, this is also idea, I will verify what we can do with this endpoint.

Reply:
Why don't we add a [wait_for_successful_query](https://docs.saltstack.com/en/latest/ref/states/all/salt.states.http.html#salt.states.http.wait_for_successful_query) at the end of [`kubelet`](https://github.com/kubic-project/salt/blob/master/salt/kubelet/init.sls#L101) in `kubelet/init.sls`, so we don't have to mess with the orchestration and a simple highstate would suffice for checking the node's health...